realis.go

```diff
@@ -256,11 +201,11 @@ func newTBinTransport(url string, timeout int, config *RealisConfig) (thrift.TTr
 // It will retry all calls which result in a temporary failure as well as calls that fail due to an EOF
 // being returned by the http client. Most permanent failures are now being caught by the thriftCallWithRetries
 // function and not being retried but there may be corner cases not yet handled.
-func NewRealisClient(options ...ClientOption) (Realis, error) {
+func NewRealisClient(options ...ClientOption) (*RealisClient, error) {
 	config := &RealisConfig{}
 
 	// Default configs
-	config.timeoutms = 10000
+	config.timeout = 10 * time.Second
```

Comment:
better make all the default timers constant so you can track where they are used